src/site/assignments/huffman-codes/huffman-codes.rst

```diff
@@ -288,12 +288,12 @@ correctly by
 Some Hints
 ==========
 
-* Work on one function at a time
-* Get each function working perfectly before you go on to the next one
-* Test each function as you write it
+* Work on one method at a time
+* Get each method working perfectly before you go on to the next one
+* Test each methog as you write it
```

Comment:
methog

intellij has _replace in files_ bruh, why you manually typing this in every time